$NetBSD: patch-ab,v 1.3 2011/04/14 03:27:17 makoto Exp $

See following mail (in Japanese)
[mule-ja-2009:09758] Re: (emacs-23.2) M-x egg-toroku-region  Invalid code(s) (Re: tamago Anthy
http://www.m17n.org/mlarchive/mule-ja/201008/msg00005.html

--- egg-edep.el.orig	2001-01-28 03:45:24.000000000 +0900
+++ egg-edep.el	2010-10-04 11:42:17.000000000 +0900
@@ -74,7 +74,10 @@
 	  (set-category-table (copy-category-table))
 	  (setq category (get-unused-category))
 	  (define-category category "")
-	  (modify-category-entry (make-char charset) category)
+	  (if (and (boundp 'emacs-major-version)
+              	   (< emacs-major-version 23))
+	      (modify-category-entry (make-char charset) category)
+	    (map-charset-chars 'modify-category-entry charset category)  )
 	  (string-match (concat "\\c" (list category) "+") string start))
       (set-category-table cur-ct))))
 
